frontend/public/module/k8s/pods.ts

```diff
@@ -84,6 +86,12 @@ export const VolumeSource = {
     label: 'Projected',
     description: 'A projected volume maps several existing volume sources into the same directory.',
   },
+  persistentVolumeClaim: {
```

Comment:
It's unclear to me why this change is needed. I'm not sure it's right since there are other more specific values for different kinds of PVCs above.